website/aws.erb

```diff
@@ -2950,6 +2950,31 @@
                         </li>
                     </ul>
                 </li>
+                <li>
+                    <a href="#">Traffic Mirror Resources</a>
```

Comment:
These resources should be included in the EC2 resources instead of a new category. 👍